src/components/modules/document/DocRequiredCreation/modules/UploadRequired/UploadRequired.interface.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <upload-required
+    :selected-document-type="selectedDocumentType"
+    @next="onNext"
+    @cancel="onCancel"
+  />
+</template>
+
+<script>
+import UploadRequired from './UploadRequired.interactor';
+import { documentsRequiredStore as documentsRequiredStoreModule } from '@/store';
+
+export default {
+  name: 'UploadRequiredInterface',
+
+  documentsRequiredStore: documentsRequiredStoreModule,
+
+  provide() {
+    const { documentsRequiredStore } = this.$options;
+    return {
+      gateway: {
+        async addDocTypeStatus({
+          documentType,
```

Comment:
In similar cases, we can describe method by `jsdoc` and use just single parameter without descruturization, because it will be originally passed to the next handler without and data transformation. @kazurus what do you think?